en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-soap-api-passthrough.md

```diff
@@ -0,0 +1,355 @@
+# Deploying a SOAP API(Pass-Through)  in Choreo Connect
```

Comment:
Extra space before in

Reply:
fixed in https://github.com/wso2/docs-apim/pull/6080/commits/48e6f33d655d7ae9595019f7b541f2128cc494a3